purchase_order_line_sequence/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=========================================
+Purchase order lines with sequence number
+=========================================
+
+Provides a new field sequence on the purchase order, which allows to manage the order of its purchase order lines.
+The sequence then is propagated to the Invoice lines and the Stock Moves.
+
+
+Usage
+=====
+
+-This module depends on the following module(s) :
+- * stock_picking_line_sequence (https://github.com/OCA/stock-logistics-workflow/)
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/10.0
```

Comment:
s/10.0/8.0

Reply:
corrected. comes with next push.

Reply:
@mathiasfrancke @bodedra Use new README by fragment system, copying https://github.com/OCA/maintainer-tools/tree/master/template/module/readme and editing or removing the needed sections. You can then run locally oca-gen-addon-readme if you have OCA/maintainer-tools installed and you want to pregenerate the README.rst file (preferred), or keep previous README.rst file.